modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-options/src/main/resources/META-INF/resources/options_field.js

```diff
@@ -56,6 +56,12 @@ AUI.add(
 					initializer: function() {
 						var instance = this;
 
+						var builder = instance.get('builder');
+
+						if (builder) {
+							builder.after('editingLanguageIdChange', instance._afterEditingLanguageIdChange.bind(instance));
```

Comment:
Hey @pedroqueiroz could you add these line of code in the eventHandlers array? For it could be removed when this field will be destroyed